docs/getting-started/serving-for-inference-and-training.md

```diff
@@ -94,3 +95,13 @@ dataset = client.training_set(name, variant).repeat(5).shuffle(1000).batch(64)
 for feature_batch, label_batch in dataset:
     # Run through a shuffled dataset 5x in batches of 64
 ```
+
+## Model Registration
+
+We can optionally include a model name at the time of serving features or training sets to create a logical grouping of models and their dependencies.
+
+```python
+fpf = client.features([("fpf", "quickstart")], {"passenger": "1"}, model="passengers_random_forest")
+```
+
+In the background, Featureform stores the model name and its associated feature(s) in metadata. The `training_set` method on the serving client accepts the same options `model` param. We can see a list of [models in the dashboard](exploring-the-feature-registry.md#models) and drill down into a specific model to see the features and/or training sets that are associated with it.
```

Comment:
Something like : Featueform tracks which models are associated being served which features and training sets. Both `features` and `training_set` accept a `model` parameter which is typically set as a string. *

"I think that the associated features(s) in metadata" would be confusing if you didn't understand how FF works in the backend